source/val/function.h

```diff
@@ -232,6 +232,14 @@ class Function {
     return function_call_targets_;
   }
 
+  BasicBlock* GetMergeHeader(BasicBlock* merge_block) {
```

Comment:
Function comments. Especially useful would be indicating when these become usable.